tests/tests_pytorch/accelerators/test_hpu.py

```diff
@@ -303,3 +303,23 @@ def training_epoch_end(self, outputs) -> None:
     trainer.fit(model)
 
     assert all(model.optims)
+
+
+@RunIf(hpu=True)
+def test_hpu_device_stats_monitor(tmpdir):
+
+    hpu_stats = HPUAccelerator().get_device_stats("hpu")
+    fields = [
+        "Limit",
+        "InUse",
+        "MaxInUse",
+        "NumAllocs",
+        "NumFrees",
+        "ActiveAllocs",
+        "MaxAllocSize",
+        "TotalSystemAllocs",
+        "TotalSystemFrees",
+        "TotalActiveAllocs",
+    ]
+    for f in fields:
+        assert any(f in h for h in hpu_stats.keys())
```

Comment:
I would test instead that it returns a list of strings and that some of the values exist (the basic ones). We don't want this to fail if the names are changed in torch_hpu

Reply:
As and when hpu package is upgraded, this will need change. Its better this way to catch the exceptions